modules/codegen/src/smithy4s/codegen/Renderer.scala

```diff
@@ -304,7 +304,9 @@ private[codegen] class Renderer(compilationUnit: CompilationUnit) { self =>
       sField: Option[StreamingField]
   ): Line = sField match {
     case Some(StreamingField(name, tpe, hints)) =>
-      val mh = if (hints.isEmpty) "" else s".addHints(${memberHints(hints)})"
+      val mh =
+        if (hints.isEmpty) Line.empty
```

Comment:
crazy idea, what would happen if we had an override for Line's `toString` that throws an exception? It should help prevent an accidental `s"..."` in the future, but it seems a bit extreme

Reply:
My only concern with this is that we may not catch the problem in testing which means it would be left up to a random user to discover the issue